pom.xml

```diff
@@ -60,6 +61,8 @@
     <test.output.tofile>true</test.output.tofile>
     <top.dir>${project.basedir}</top.dir>
 
+    <jdk.version>1.7</jdk.version>
```

Comment:
Do we need to be supporting Java7 for anything? Can we move to 8?

Reply:
There are significant changes in the java code between phoenix-hive and phoenix-hive, it is not just a copy and pom.xml change. 
Hive 3.x supports Hadoop 3 but hive 2 does not, Hive3 is intended to be used with Phoenix 5 meanwhile the old phoenix-hive connector is working with phoenix 4.

This JDK version is in connectors pom.xml, all the modules but the phoenix-hive3 uses jdk 1.7. In phoenix-hive3 it is being overwritten to 1.8

Reply:
Ok, cool. I didn't have an easy way to validate this. Copying is probably easiest! Thanks!